canvas.js

```diff
@@ -2256,3 +2258,8 @@ if (typeof Path2D !== 'function' ||
     Path2D = Path_;
   })();
 }
+
```

Comment:
Without the `undefined` guards, the script won't load - for me - in a browser, as `require` is not defined. That is, without the guards, the tests will fail.